test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -45,9 +45,9 @@ public IModuleConfigBuilder AddEdgeHub(string image = null, bool optimizeForPerf
             return builder;
         }
 
-        public IModuleConfigBuilder AddModule(string name, string image)
+        public IModuleConfigBuilder AddModule(string name, string image, bool shouldRestart = true)
```

Comment:
I am defaulting the restart policy to "always" as it is the intention for the ~20 occurrences where this function is called.